elasticsearch/sgconfig/roles.yml

```diff
@@ -161,3 +161,17 @@ sg_project_operations:
         - indices:admin/mappings/fields/get*
         - indices:admin/validate/query*
         - indices:admin/get*
+
+# To support multi-tenancy. User's access to indices is restricted to indices belonging to the user's projects, enforced by DLS.    
+project_user:    
+  readonly: true    
```

Comment:
I think we will also need these cluster permissions: https://github.com/opendistro-for-elasticsearch/security/blob/v0.10.0.4/securityconfig/roles.yml#L51-L52

Reply:
added